### PR TITLE
docs: link to experimental CMS quickstarts

### PR DESCRIPTION
**Pull request by jeroenpeters1986**
Added links to the Quickstarts from the experiment page

## The Issue
As I stated in issue
* #5171, I overlooked the quickstarts the first time, so I put helpful links

## How This PR Solves The Issue
It adds links

## Manual Testing Instructions
None

## Automated Testing Overview
No tests needed for docs

## Related Issue Link(s)
As I stated in issue#5171, I overlooked the quickstarts the first time, so I put helpful links

## Release/Deployment Notes
None

### CODE REVIEW

**Comment by jeroenpeters1986** [comment]
@rfay I'm sorry, I copied the content of the PR but forgot to edit the title :-( I cant seems to run a re-check for the title myself

**Comment by rfay** [comment]
The failure seems to be markdownlint now, not the title, https://github.com/ddev/ddev/actions/runs/7232772716/job/19707303814?pr=5638

**Review by rfay — APPROVED** [review]
I'm so glad you added this!

**Comment by jeroenpeters1986** [comment]
Thanks, glad to do so!Op 16 dec 2023 om 16:45 heeft Randy Fay ***@***.***> het volgende geschreven:﻿
@rfay approved this pull request.

I'm so glad you added this!

File: docs/content/users/configuration/experimental.md [hunk 1]
```diff
@@ -6,4 +6,6 @@ DDEV v1.22+ supports Python-based projects, including those built with Django 4
 
 `ddev config --project-type=django4` will by default a project to use the `nginx-gunicorn` `webserver_type` and the `postgres` database type.
 
+To get started with either one of these Python projects, checkout the [Django4 Quickstart](../quickstart/#django-4-experimental) or the [Flask Quickstart](../quickstart/#pythonflask-experimental).
```
Comment (mattstein) [inline docs/content/users/configuration/experimental.md:9]: This needs a space to be consistent with the page it links to and [Django’s own formatting](https://docs.djangoproject.com/en/4.1/releases/4.0/): 

```suggestion
To get started with either one of these Python projects, checkout the [Django 4 Quickstart](../quickstart/#django-4-experimental) or the [Flask Quickstart](../quickstart/#pythonflask-experimental).
```

**Review by mattstein — CHANGES_REQUESTED** [review]
I’d recommend adding a space, but otherwise looks good to me!

**Comment by jeroenpeters1986** [comment]
Sorry for all the commits and runs guys, but it's finally ok :-D

**Review by stasadev — APPROVED** [review]

**Pull request merged**